src/commands/staff/Leaderboard.ts

```diff
@@ -0,0 +1,306 @@
+import {
```

Comment:
Does this command just generate a static leaderboard (i.e. a leaderboard embed that doesn't change, even after new leaders are added or existing leaders add more runs)?

If this is the case, and you expect the public to see this, you might consider having the bot send a separate message containing the embed (right now, the bot just edits its interaction, which means people can see the command that was executed).

Reply:
This generates the leaderboard each time the command is called.  Like Ooga.

Reply:
Deciding against this, since it adds complexity to the dungeon selection panel.  This is the same way that getstats and finduser are implemented so I don't see a reason to change it.

Reply:
Oh I see what you meant. I thought this was displaying the leaderboard to everyone (like in a leaderboard channel). Makes sense.